wallet/src/Pos/Wallet/Web/Methods/History.hs

```diff
@@ -77,7 +78,8 @@ getHistory
     -> m (Map TxId (CTx, POSIXTime), Word)
 getHistory cWalId accIds mAddrId = do
     -- FIXME: searching when only AddrId is provided is not supported yet.
-    accAddrs <- S.fromList . map cwamId <$> concatMapM (getAccountAddrsOrThrow Ever) accIds
+    accAddrs <- S.fromList . map cwamId <$>
+                concatMapM (getAccountAddrsOrThrow Ever (NeedSorting False)) accIds
```

Comment:
Is this `fromList` fast enough? Won't `par` be useful here as well?

Reply:
Nice point